service/src/main/java/com/epam/esm/services/constants/TagSearchParameters.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.esm.services.constants;
+
+public enum TagSearchParameters {
+    NAME;
+
+    public static TagSearchParameters getEntryByParameter(String s) {
```

Comment:
the same. Duplication code.

Reply:
Don't know how to resolve it properly due to absence of inheritance ii enums